tools/releaseBuild/azureDevOps/releasePipeline.yml

```diff
@@ -78,6 +78,31 @@ stages:
       globalToolExeName: 'pwsh'
       globalToolPackageName: 'PowerShell.Linux.x64'
 
+- stage: ValidateFxdPackage
+  displayName: Validate Fxd Package
+  dependsOn: []
+  jobs:
+  - template: templates/release-FxdPackage.yml
+    parameters:
+      jobName: WinFxdPackage
+      displayName: Fxd Package Test Win
+      imageName: windows-latest
+      packageNamePattern: '**/*win-fxdependent.zip'
+
+  - template: templates/release-FxdPackage.yml
```

Comment:
Can we include something about validation in the name of the file?

Reply:
ok I will update the file name

Reply:
Updated